libraries/chain_kv/bytes.cpp

```diff
@@ -0,0 +1,119 @@
+#include <string_view>
+
+#include <boost/pool/pool.hpp>
+
+#include <b1/session/bytes.hpp>
+
+using namespace b1::session;
+
+const bytes bytes::invalid{};
```

Comment:
invalid is just a representation of a bytes instance where all data members are null which is how one of these types is default constructed.  Since the default constructor is hidden, I needed a way for a user to compare a bytes instance provided to them against what is being defined as an invalid instance.